plugin.video.montreal.greek-tv/addon.py

```diff
@@ -18,7 +18,9 @@
         along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-
+import xbmc
 from resources.lib.default import plugin 
 
+xbmc.executebuiltin("InhibitScreensaver(true)")
+xbmc.executebuiltin("InhibitIdleShutdown(true)")
```

Comment:
This is a definite no-go. If you want to disable the screensaver while your addon is running then you can use `try/finally` clause and restore everything as it was in `finally` block. Also an option in your addon setting would be desirable to control this behavior (disabling the screensaver/idle power off).

Reply:
I made the change .
 I couldn't find the way  as you suggested but added condition that : inhibit screensaver/idle power off is set to false when no playing streaming. 
Also added the settings so the user can disable  screensaver/idle power off.